list.go

```diff
@@ -188,8 +223,16 @@ func (p *Popularity) RequestReposForPageToken(pagetoken string) (PopularList, er
 
 	// Since `IsWriteToFile` is false so it **doesn't** call `WriteToFile`
 	if p.IsWriteToFile {
-		//writing the reponse in ./popularity/namespace/currentfilename.json
-		p.WriteToFile(resp.Body(), p.currentFilename)
+		//writing the reponse in ./popularity/namespace/currentfileName.json
+		p.WriteToFile(resp.Body(), p.currentFileName, p.fileNamePath)
+		if err != nil {
```

Comment:
From where we will get `err` value??

Reply:
![image](https://user-images.githubusercontent.com/46858011/91956289-17ab6180-ed22-11ea-8cad-d47b959cb4b4.png)
Actually I have initially sent the PR to quay, but then I have noticed this issue. That's why I have written it correctly in growth metrics, and also changed it in quay but I haven't pushed it. So don't worry this is resolved(I have added the code to git. So in the next commit to quay it will be added)
Right now is it causing any errors? If yes then you can replace this
```
if p.IsWriteToFile {
		// writing the reponse in ./popularity/namespace/currentfilename.json
		err := p.WriteToFile(resp.Body(), p.currentFileName, p.fileNamePath)
		if err != nil {
```
i.e only add err:= in front of the line. 
Otherwise, I will push this with correcting some other minor comment mistakes next time.

Reply:
no, it's not causing any error

Reply:
please make sure to resolve in next PR :)